core-bundle/src/EventListener/MakeResponsePrivateListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+
+class MakeResponsePrivateListener
+{
+    /**
+     * Make sure the current response becomes a private response if any of the following
```

Comment:
What if the request was not cacheable? Then it should not become private, right? Can we use `$response->isCacheable()` ?

Reply:
If it's not cacheable we can ignore it anyway. Added in 66ee53af766e67ccb270488539d1399e5893541c.